### PR TITLE
Changing Arch WiFi iso to newer build

### PR DESCRIPTION
**Pull request by NoaHimesaka1873**

### CODE REVIEW

**Comment by networkException** [comment]
I don't really like random websites to get binaries from, can you rather open a pull request with the code changes on @Redecorating s repo?

**Comment by NoaHimesaka1873** [comment]
@networkException
Hmm... pushing directly to Redo's repo isn't that good because the firmware files... *coughs*
https://github.com/NoaHimesaka1873/archiso-t2 I uploaded a reproducible archiso repo so this might do the job?

File: docs/distributions/arch/installation.md [hunk 1]
```diff
@@ -7,7 +7,7 @@ You will need:
 
 - USB drive with at least 1GB
 - A way to plug it into your Mac (USB-C isn't USB-A)
-- A wired internet connection (i.e. USB-C to Enternet dongle) or wifi. If you need to install via wifi, you may use [this iso](https://github.com/Redecorating/archiso-mbp/releases), which has everything needed to follow the [wifi guide](https://wiki.t2linux.org/guides/wifi) and use wifi in the live environment.
+- A wired internet connection (i.e. USB-C to Enternet dongle) or wifi. If you need to install via wifi, you may use [this iso](https://mirror.funami.tech/archiso-mact2/), which has everything needed to follow the [wifi guide](https://wiki.t2linux.org/guides/wifi) and use wifi in the live environment.
```
Comment (Redecorating) [inline docs/distributions/arch/installation.md:10]: Could you change this link to your repo's releases page? https://github.com/NoaHimesaka1873/archiso-t2/releases

Comment (NoaHimesaka1873) [reply docs/distributions/arch/installation.md]: Just did that!

File: docs/distributions/arch/installation.md [hunk 2]
```diff
@@ -7,7 +7,7 @@ You will need:
 
 - USB drive with at least 1GB
 - A way to plug it into your Mac (USB-C isn't USB-A)
-- A wired internet connection (i.e. USB-C to Enternet dongle) or wifi. If you need to install via wifi, you may use [this iso](https://github.com/Redecorating/archiso-mbp/releases), which has everything needed to follow the [wifi guide](https://wiki.t2linux.org/guides/wifi) and use wifi in the live environment.
+- A wired internet connection (i.e. USB-C to Enternet dongle) or wifi. If you need to install via wifi, you may use [this iso](https://github.com/NoaHimesaka1873/archiso-t2/releases), which has everything needed to follow the [wifi guide](https://wiki.t2linux.org/guides/wifi) and use wifi in the live environment.
```
Comment (Redecorating) [inline docs/distributions/arch/installation.md:10]: > which has everything needed to follow the [wifi guide](https://wiki.t2linux.org/guides/wifi) and use wifi in the live environment.

does your iso have the firmware automatically selected? If so then they won't need to follow the wifi guide to get wifi working in the live environment

Comment (NoaHimesaka1873) [reply docs/distributions/arch/installation.md:10]: Yeah, my iso automatically selects firmware and also contains kernels and firmwares as packages for pacstrap. I think we need different guide to guide users to install those packages when pacstraping.

**Comment by NoaHimesaka1873** [comment]
fixed syntax error.

**Comment by NoaHimesaka1873** [comment]
wtf markdown lint? why does it care about how much spaces i use?

**Comment by NoaHimesaka1873** [comment]
finally, with the help of VSCode, linted

**Pull request merged**